examples/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "tinyman-js-sdk-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start:dev": "nodemon"
+  },
+  "author": "",
```

Let's leave there a sign :D

```suggestion
  "author": "Tinyman Core Team",
```